### PR TITLE
attempt at command layout

### PR DESCRIPTION
**Pull request by coinzdude**
struggling here with the layout to use the horizontal correctly

### CODE REVIEW

File: packages/selenium-ide/src/browser/windows/ProjectEditor/tabs/Tests/CommandFields/LocatorField.tsx [hunk 1]
```diff
@@ -19,7 +22,20 @@ const CommandLocatorField: FC<CommandArgFieldProps> = ({
...
   const fieldNames = (fieldName + 's') as PluralField
   const FieldName = capitalize(fieldName)
-  const commandData = commands[command.command]
+
+  const fullnote = commands[command.command][fieldName]?.description ?? ''
+  let note = new String()
+  let longNote = ''
+  let isLongNote = false
+
+  const truncateLength = 60
```
Comment (toddtarsi) [inline packages/selenium-ide/src/browser/windows/ProjectEditor/tabs/Tests/CommandFields/LocatorField.tsx]: @coinzdude - Don't do this in code. This is one line of css: `text-overflow: ellipsis`

File: packages/selenium-ide/src/browser/windows/ProjectEditor/tabs/Tests/CommandFields/LocatorField.tsx [hunk 2]
```diff
@@ -1,9 +1,12 @@
+import { HelpCenter } from '@mui/icons-material'
 import AddToHomeScreenIcon from '@mui/icons-material/AddToHomeScreen'
-import Autocomplete from '@mui/material/Autocomplete'
-import FormControl from '@mui/material/FormControl'
-import FormHelperText from '@mui/material/FormHelperText'
-import IconButton from '@mui/material/IconButton'
-import TextField from '@mui/material/TextField'
+import {
```
Comment (toddtarsi) [inline packages/selenium-ide/src/browser/windows/ProjectEditor/tabs/Tests/CommandFields/LocatorField.tsx]: Please use the above import pattern for these files. It tree shakes unused components, providing notable performance benefits if done everywhere.

**Comment by coinzdude** [comment]
will do! I wasn't sure why they were not combined, makes sense.

File: packages/selenium-ide/src/browser/windows/ProjectEditor/tabs/Tests/CommandFields/LocatorField.tsx (hunk 1 above)
Comment (coinzdude) [reply packages/selenium-ide/src/browser/windows/ProjectEditor/tabs/Tests/CommandFields/LocatorField.tsx]: great!

**Review by coinzdude — COMMENTED** [review]
Looks good, will test

**Comment by toddtarsi** [comment]
Closing this, table view is in

**Pull request closed without merging**